src/getCalculatedMeasures.ts

```diff
@@ -31,7 +22,28 @@ export const getCalculatedMeasures = async (
                 cmEntry.entry.content
               );
 
-              return result.calculatedMember["$"];
+              const { owners, readers, isDirectory, canRead, canWrite } =
+                cmEntry.entry;
+
+              const calculatedMeasure = {
+                owners,
+                readers,
+                isDirectory,
+                canRead,
+                canWrite,
```

Comment:
`canRead` and `canWrite` are generated on the fly when fetching a file from the content server for a given user.
It would make no sense to define it statically on the content server.
And so it's not part of the [API](https://artifacts.activeviam.com/documentation/rest/6.0.0-M1/content-server.html#files___import_true_post) when you import an entry: you just have owners and readers. 
So you just discard those attributes here.

Let me know if something is unclear.